styles.css

```diff
@@ -58,18 +63,207 @@
   align-items: center;
 }
 
-.connect > ul {
+.connect-ul {
   list-style: none;
   display: flex;
   margin-right: 220px;
 }
 
-.connect > ul > li {
+.connect-ul > li {
   padding: 6px;
   margin-top: -16px;
 }
 
-.connect > ul > li,
+.connect-ul > li,
 .hamburgerIcon > img {
   cursor: pointer;
 }
+
+.works-section {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  gap: 88px;
+  margin-top: 114px;
+}
+
+.work {
+  width: 327px;
+  height: 578px;
+  margin-left: 24px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+}
```

Comment:
- [ ] KIndly make sure each project card content is aligned on the page using the CSS `flexbox` in order to meet the project requirements. Not only that, CSS `flexbox` will help in the future to maintain easily page responsiveness. Kindly read [this lesson](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox) for more information.

- [ ] KIndly make sure to give the project card the same `background-color` as the one on the provided Figma design.

    - ### YOUR DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/228072263-8668deff-2901-43ac-9e52-fd3d639a2ca5.png)

    - ### FIGMA DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/228072366-87d032b9-2109-4e94-85eb-62ccc5dc6368.png)